cpp/tests/groupby/sort/single_column_groupby_test.cuh

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
This file is redundant with what is used in the hash groupby tests.

Reply:
No it is not, hash based groupby need to sort values before compare them, sort-based groupby not. And due to the sort-based groupby use std::vector<gdf_column> instead of cudf::table as output values, This file is needed.

Reply:
Okay, then it is 50% redundant: `sort_by_key`, `expect_tables_are_equal`, and related functions are identical. Eliminate this redundancy.